datafusion/functions/src/unicode/translate.rs

```diff
@@ -72,6 +75,7 @@ impl ScalarUDFImpl for TranslateFunc {
 
     fn invoke(&self, args: &[ColumnarValue]) -> Result<ColumnarValue> {
         match args[0].data_type() {
+            DataType::Utf8View => make_scalar_function(translate::<i32>, vec![])(args),
```

Comment:
I suspect this I think you'll likely also need to change the implementation of `translate` as well (see for example what @PsiACE  has done in https://github.com/apache/datafusion/pull/11970)

Reply:
Sounds good I'll take a look 👍